### PR TITLE
PQ-8::Add dateChanged in the PatientQueueMapper

### PR DESCRIPTION
**Pull request by slubwama**
https://issues.openmrs.org/browse/PQ-8

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you add some description to the ticket?

File: api/src/main/java/org/openmrs/module/patientqueueing/mapper/PatientQueueMapper.java [hunk 1]
```diff
@@ -35,6 +35,8 @@
 	private String status;
 	
 	private String dateCreated;
+
+	private String dateChanged;
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/patientqueueing/mapper/PatientQueueMapper.java:39]: Don't you want to also know who changed?

Comment (slubwama) [reply api/src/main/java/org/openmrs/module/patientqueueing/mapper/PatientQueueMapper.java:39]: Not really. But let me add it.

Comment (slubwama) [reply api/src/main/java/org/openmrs/module/patientqueueing/mapper/PatientQueueMapper.java:39]: Done responding to this comment

**Pull request merged**